packages/material-ui/src/NativeSelect/NativeSelectInput.js

```diff
@@ -84,7 +84,7 @@ export const nativeSelectIconStyles = ({ styleProps, theme }) => ({
   // to the input and to support wrapping tags..
   position: 'absolute',
   right: 0,
-  top: 'calc(50% - 12px)', // Center vertically
+  top: 'calc(50% - .5em)', // Center vertically
```

Comment:
maybe to clarify how it works?

```suggestion
  top: 'calc(50% - 0.5em)', // Center vertically, height is 1em
```